portal-web/test/functional/com/liferay/portalweb/paths/portlet/marketplace/AppManager.path

```diff
@@ -26,6 +26,97 @@
 	<td></td>
 	<td></td>
 </tr>
+<!--APP_STORE-->
+<tr>
+	<td>APP_ANY</td>
+	<td>//div[contains(@data-se,'o-form-label') and contains(.,'${key_text}')]//following-sibling::div[contains(@class,'o-form-input')]//input[contains(@type,'text')] | //div[contains(@data-se,'o-form-label') and contains(.,'${key_text}')]//following-sibling::div[contains(@class,'o-form-input')]//input[contains(@type,'password')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_FILTER_LABEL</td>
+	<td>//span[contains(@class,'aui-field-select')]/span[contains(@class,'aui-field-content')]/label[contains(.,'${key_filterName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_FILTER_SELECT</td>
+	<td>//span[contains(@class,'aui-field-content')]/label[contains(.,'${key_filterName}')]//following-sibling::span[contains(@class,'aui-field-element')]/select</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_FILTER_SELECT_OPTION</td>
+	<td>//span[contains(@class,'aui-field-content')]//select/option[contains(.,'${key_filterBy}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_PURCHASE</td>
+	<td>//div[contains(@class,'purchase')]/a[contains(.,'${key_type}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_PURCHASE_SUBMIT</td>
+	<td>//span[contains(@class,'aui-button-content')]/input[contains(@type,'submit')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_PURCHASE_AGREE_CHECKBOX</td>
+	<td>//span[contains(@class,'aui-field-element')]/input[contains(@type,'checkbox')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_PURCHASE_INFO</td>
+	<td>//div[contains(@class,'container')]/h3[contains(.,' Thank You! ')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_PURCHASE_USER_SELECT</td>
+	<td>//div[contains(@class,'card user')]/div[contains(.,' For Personal Use Only ')]//following-sibling::div[contains(@class,'card-action')]/a[contains(.,'Select')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_RESULT_FILTER</td>
+	<td>//div[contains(@class,'marketplace-entry-tile')]/div//following-sibling::ul/li[contains(.,'${key_filterName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_RESULT_BOX</td>
+	<td>//span[contains(@class,'aui-field-element')]/input[contains(@class,'aui-field-input-text')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_TIPY</td>
+	<td>//div[contains(@class,'marketplace-entry-tile')]//a[contains(.,'${key_appType}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APP_TITLE</td>
+	<td>//div[contains(@class,'marketplace-entry-tile')]//a[contains(.,'${key_searchName}')] | //div[contains(@class,'app-entry-display')]//a[contains(.,'${key_searchName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MARKETPLACE_TITLE</td>
+	<td>//div[contains(@class,'banner-content')]/a[contains(@title,'liferay-marketplace')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MARKETPLACE_DESCRIPTION</td>
+	<td>//div[contains(@class,'authorize')]/p</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SEARCH_BUTTON</td>
+	<td>//*[@data-qa-id='searchButton'] | //button[contains(.,'Search')] | //input[contains(@value,'Search')]</td>
```

Comment:
Only leave the one we used in our tests.

Reply:
AppManager#SEARCH_BUTTON leave last one. Still tracking cause maybe other testcases will use it